infection_monkey/monkey.py

```diff
@@ -216,7 +228,8 @@ def cleanup(self):
         self._keep_running = False
 
         # Signal the server (before closing the tunnel)
-        ControlClient.send_telemetry("state", {'done': True})
+        if not self._upgrading_to_64:
```

Comment:
We need to rework this. If we are upgrading to 64, the only thing we _currently_ did is add a firewall rule, which we anyway don't clean up :(
I'd rather do a case for upgrading rather than have this if copied in other places in this function.

Reply:
Changed it. I'm not sure it's better.